src/Configurations/Fields/Sections/Synchronization.js

```diff
@@ -52,6 +64,8 @@ export const Synchronization = ({ isNonInteractive }) => {
                   type="number"
                   name="accessionDelay"
                   hasClearIcon={false}
+                  format={formatNumber}
```

Comment:
In the final-form example they use `parse`, not `format`
https://codesandbox.io/s/no20p7z3l?file=/index.js:1090-1139